### PR TITLE
Typos in libtransmission

### PR DESCRIPTION
**Pull request by GaryElshaw**
One of a few potential typo fixes in libtransmission

### CODE REVIEW

File: libtransmission/tr-utp.cc [hunk 1]
```diff
@@ -61,7 +61,7 @@ struct UTPSocket* utp_create_socket(struct_utp_context* /*ctx*/)
     return nullptr;
 }
 
-int utp_connect(UTPSocket* /*socket*/, sockaddr const* /*to*/, socklen_t /*tolen*/)
+int utp_connect(UTPSocket* /*socket*/, sockaddr const* /*to*/, socklen_t /*token*/)
```
Comment (ckerr) [inline libtransmission/tr-utp.cc:64]: not a typo; note that `tolen` is of type `socklen_t` and is adjacent to a param named `to` :smile_cat:

Comment (ckerr) [reply libtransmission/tr-utp.cc:64]: marked resolved by accident? Here and below still need to be `tolen` not `token`

**Review by ckerr — CHANGES_REQUESTED** [review]
token -> tolen

File: libtransmission/tr-utp.cc (hunk 1 above)
Comment (GaryElshaw) [reply libtransmission/tr-utp.cc]: Sorry about that, fixed now.

**Review by ckerr — APPROVED** [review]

**Pull request merged**